client/buttonPress.py

```diff
@@ -0,0 +1,27 @@
+import pygame as pg
+
+pg.init()
+COLOR = pg.Color('red')
```

Comment:
constants should be in definition.py

try to use hex tuples for colors ie. `RED = (255,0,0)`